src/CONST.ts

```diff
@@ -965,13 +966,14 @@ const CONST = {
         MAX_RETRY_WAIT_TIME_MS: 10 * 1000,
         PROCESS_REQUEST_DELAY_MS: 1000,
         MAX_PENDING_TIME_MS: 10 * 1000,
+        REACHABILITY_TIMEOUT_MS: 60 * 1000,
```

Comment:
Actually, why _timeout_? Timeout is the time until we give up some request or operation.

What about `BACKEND_REACHABILITY_CHECK_INTERVAL_MS` ? If this name feels too long, it could be shortened to `BACKEND_CHECK_INTERVAL_MS`.

Reply:
@tienifr Bump on this minor comment